src/App.js

```diff
@@ -3,7 +3,7 @@ import { Route, withRouter, Switch } from 'react-router-dom';
 import RegistrationPage from './components/auth-components/registration-page';
 import LoginPage from './components/auth-components/login-page';
 import Lists from './components/Lists';
-import Items from './components/Items';
+import Items from './components/Items/index';
```

Comment:
leave this as `import Items from './components/Items'`

Reply:
for some reason i'm getting an error when i do that. any idea why that would ??